survey-sdk/src/main/java/co/nimblehq/survey/sdk/entity/MetaEntity.kt

```diff
@@ -0,0 +1,17 @@
+package co.nimblehq.survey.sdk.entity
+
+import com.squareup.moshi.Json
+
+data class MetaEntity(
+    @Json(name = "page")
+    val page: Int,
+
+    @Json(name = "pages")
+    val pages: Int,
+
+    @Json(name = "page_size")
+    val pageSize: Int,
+
+    @Json(name = "records")
+    val records: Int
```

Comment:
@hoangnguyen92dn agree with this

Reply:
@hoangnguyen92dn I updated with nullable, but if I change the annotation from `Json` to `field:Json`,  the property `pageSize` will not be parsed. You can can checkout current branch and run the test case to see.

Reply:
@doannimble Hmm. It's weird. I tried to rename `pageSize` to `page_size` with `field:Json` then it works 🤔

Reply:
@hoangnguyen92dn Yes, if I renamed to `page_size`, it will work. But it does not match with `coding convention`. So I suggest this is `nice to have`

Reply:
@doannimble I believe the issue comes from somewhere else from the Moshi parser. Plz see on their repo if there is the same reported issue. Then we could add the link of the issue for reference.

Reply:
@hoangnguyen92dn The MetaEntity is not included in `moshi-json-api`. Only the Class which included in `moshi-json-api` need to use the annotation `@field:Json`. 
With normal `moshi-adapter`, we only need to use `@Json`.
So, I think I'm doing correct implementation, no need to make change.
What do you think?

Reply:
@doannimble Both `@Json` and `@field:Json` comes from `com.squareup.moshi.Json`. If it depends on the annotation so how do the other fields still working perfectly? We should know the root cause of issue instead of skipping it before applying for the another working solution 🙏

Reply:
@hoangnguyen92dn it because of the `json-api' lib mapping by `field-json` annotation
`public class MoshiJsonNameMapping implements JsonNameMapping {
    @Override
    public String getJsonName(Field field) {
        String name = field.getName();
        Json json = field.getAnnotation(Json.class);
        if (json != null) {
            name = json.name();
        }
        return name;
    }
}`
And the `moshi adapter` we use for parsing MetaEntity is not from `json-api` lib.
That's the reason why we use the different annotation for the entities included and excluded from `json-api`.
Is it OK now ?

Reply:
@doannimble As we discussed, we could leave it as it is for now. Please add the comment to describe the reason why do we need to use `@Json` instead of `@field:Json` to address the issue and we will the solution later.

Reply:
@hoangnguyen92dn I've left the comment on the MetaEntity